app/src/components/MakeTimer.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {useState} from "react"
+
+
+interface makeTimerProps {
+    onSubmit: any
+}
+
+export default function MakeTimer(props: makeTimerProps) {
+
+  const [timerLength, setTimerLength] = useState(25)
```

Comment:
Probably add these values to localStorage in the future so they don't have to set it each time